src/input/controls.rs

```diff
@@ -61,10 +61,7 @@ pub fn handle_keypress_loading_mode<B: Backend>(evt: Event, app: &mut App<B>) {
         key!(char '\n') => {
             app.handle_enter();
         }
-        key!(Delete) => {
-            app.show_warning_modal();
```

Comment:
We still need this. It's a new feature where we show a warning modal if we try to delete while loading.
Sorry there are no tests for this... I haven't yet figured out how to test "loading mode" properly. :(

Once you fix this (I think just bringing this back and changing it to Backspace should be enough) we can merge.

Reply:
I will fix it later. I was having trouble with merge conflicts, maybe I accidentally removed it.